semana18/projeto-cookenu/src/database/migrations/createTables.ts

```diff
@@ -0,0 +1,40 @@
+import connection from "../connection";
+
+const createTables = async (): Promise<void> => {
+  try {
+    await connection.raw(`
+        CREATE TABLE Users_Cookenu (
```

Comment:
Uma boa aqui é adicionar o IF NOT EXISTS junto do create table, para evitar de tomar um erro de **tabela já existente**